src/main/java/com/jayway/maven/plugins/android/phase_prebuild/ClasspathModifierLifecycleParticipant.java

```diff
@@ -285,5 +292,25 @@ private Dependency createSystemScopeDependency( Artifact artifact, File location
         dependency.setSystemPath( location.getAbsolutePath() );
         return dependency;
     }
+    
+    private Dependency findProvidedDependencies( Dependency dexDependency, MavenProject project ) 
+    {
+        for ( Dependency dependency : project.getDependencies() ) 
+        {
+            if ( dependency.getScope().equals( Artifact.SCOPE_PROVIDED ) ) 
```

Comment:
> Are they always going to be SCOPE_PROVIDED?
HRM... Scope provided was put in there before because in the 3.x plugin series Maven would unpack both the jar and the library and then Android would blow up with conflicts.  If we just always remove it then it would remove the *need* for provided.  I like provided in the pom because it makes what is happening explicit, but I don't think it is necessary to check for it.

> What dependency are you looking for the AAR or it's contained JAR?
Neither.  I am looking for a jar dependency that will conflict with the system jar dependency which is about to be created.

I'll update to explicitly look for jar types.
I'm not sold on removing the provided type requirement.